reana_job_controller/slurmcern_job_manager.py

```diff
@@ -117,6 +128,7 @@ def execute(self):
             port=SlurmJobManagerCERN.SLURM_HEADNODE_PORT,
         )
         self._transfer_inputs()
+        self._pull_image()
```

Comment:
*suggestion:* What about moving `_pull_image()` before `_transfer_inputs()`? In case the Docker image is incorrect we will get errors early with no need to transfer files to Slurm first.